flux-middle/src/rty/expr.rs

```diff
@@ -22,12 +22,61 @@ use crate::{
 
 pub type Expr = Interned<ExprS>;
 
-#[derive(Clone, PartialEq, Eq, Hash, TyEncodable, TyDecodable)]
+#[derive(Clone, PartialEq, Eq, Hash, TyEncodable, TyDecodable, Debug)]
```

Comment:
I think we want the `Pretty` impl for debugging

```suggestion
#[derive(Clone, PartialEq, Eq, Hash, TyEncodable, TyDecodable)]
```